### ISSUE TITLE
Grouping the result latex equation

### ISSUE THREAD

**Issue by taoari** [seebk/LaTeXText#22]
Is it possible to group the resulting LaTeX equation? It is super inconvenient with separate symbols. For example, with rendering $x_l$, I got x and l, in order to move them, I have to manually select x and then l. It will be super convenient if they are grouped by default.

**Comment by seebk** [comment]
Good idea, makes sense and that should not be too difficult to implement. 
Will keep this open as a reminder, but cannot give any promises when I can 
do it.

**Comment by seebk** [comment]
Mhm... I just tried it and every Inkscape text element will be one SVG group after rendering. So all components of one equation should be in a group. Can you send me an example where the elements are not grouped together?
